doc/user/user/pipelines/index.md

```diff
@@ -18,48 +18,41 @@ Adding samples to the cart
 
 The cart is a place where you can temporarily store a selection of samples from across multiple projects for submission to a pipeline for analysis.
 
-You can see the current contents of the cart by clicking on the "Cart" button, on the top, right-hand side of the [dashboard](../dashboard):
-
-![Cart button location.](images/cart-button.png)
-
-The cart will appear on the right-hand side of the page:
-
-![Empty cart.](images/empty-cart.png)
-
 The cart is initially empty. You can add samples to your cart by navigating to the [samples list for a project](../samples/#viewing-samples-in-a-project) and [selecting the samples that you want to add](../samples/#selecting-samples). Once you've selected the samples that you want to add, click on the "Add to Cart" button, just above the samples list:
 
 ![Add to cart button.](images/add-to-cart-button.png)
 
-Once you've added some samples to the cart, if you click the "Cart" button on the dashboard, you'll be able to review all of the samples that you've selected, organized by project:
+Once you’ve added some samples to the cart, if you click the `Cart` button on the main navigation, you will be taken to the cart page.
 
-![Full cart.](images/full-cart.png)
+![Pipeline Cart.](images/pipeline-launch-cart.png)
 
-You can remove individual samples or all selected samples from a project by clicking on the <img src="images/remove-icon.png" class="inline" alt="Remove icon."> icon, or you can empty the cart entirely by clicking on the "Clear" button, in the top, right-hand corner of the cart:
+You can remove individual samples or all selected samples from a project by clicking on the sample menu icon:
 
-![Cart clear button.](images/cart-clear-button.png)
+ ![Cart clear button.](images/menu-icon.png)
+ 
+And either selection “Remove Sample” which will remove just a single sample or “Remove Project” which will remove all samples that belong to the same project. You can also empty the cart entirely by clicking on the “Empty” button, in the top, right-hand corner of the cart.
```

Comment:
either **selection** -> either **selecting**

Reply:
Updated in 88ad188be78219b0dca0730d47971f171daa3a18